src/components/lab/SpectrumBar.js

```diff
@@ -143,6 +143,7 @@ const SpectrumBar = ({
           (SPECTRUM_BAR_HEIGHT - SPECTRUM_BAR_LABELS_FONT_SIZE) / 2
         }
         fill={ULTRAVIOLET_BAR_LABEL_COLOR}
+        align="right"
```

Comment:
I checked and `align="right"` doesn't seem to be affecting the position of this Text element. Would just remove...

Reply:
It's true that alone `align=true` does not do much alone. But together with `width` it allows for the text to be more "dynamic" and always aligned to the right (so adding more or less characters won't lead to adjust some factor constant). I did some small refactors around it so it works fully!